[2022-10-10 19:13:31,988] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: DesafioAirflow.task_1 scheduled__2022-10-09T22:13:29.000546+00:00 [queued]>
[2022-10-10 19:13:31,993] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: DesafioAirflow.task_1 scheduled__2022-10-09T22:13:29.000546+00:00 [queued]>
[2022-10-10 19:13:31,994] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-10-10 19:13:31,994] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-10-10 19:13:31,994] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-10-10 19:13:32,005] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): task_1> on 2022-10-09 22:13:29.000546+00:00
[2022-10-10 19:13:32,010] {standard_task_runner.py:52} INFO - Started process 23361 to run task
[2022-10-10 19:13:32,014] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'DesafioAirflow', 'task_1', 'scheduled__2022-10-09T22:13:29.000546+00:00', '--job-id', '7', '--raw', '--subdir', 'DAGS_FOLDER/example_desafio.py', '--cfg-path', '/tmp/tmpdjybk4ei', '--error-file', '/tmp/tmpxulrcsc0']
[2022-10-10 19:13:32,015] {standard_task_runner.py:77} INFO - Job 7: Subtask task_1
[2022-10-10 19:13:32,066] {logging_mixin.py:109} INFO - Running <TaskInstance: DesafioAirflow.task_1 scheduled__2022-10-09T22:13:29.000546+00:00 [running]> on host LAPTOP-P5116HF5.localdomain
[2022-10-10 19:13:32,117] {taskinstance.py:1424} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=airflow@example.com
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=DesafioAirflow
AIRFLOW_CTX_TASK_ID=task_1
AIRFLOW_CTX_EXECUTION_DATE=2022-10-09T22:13:29.000546+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-10-09T22:13:29.000546+00:00
[2022-10-10 19:13:32,141] {python.py:175} INFO - Done. Returned value was: None
[2022-10-10 19:13:32,146] {taskinstance.py:1267} INFO - Marking task as SUCCESS. dag_id=DesafioAirflow, task_id=task_1, execution_date=20221009T221329, start_date=20221010T221331, end_date=20221010T221332
[2022-10-10 19:13:32,187] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-10-10 19:13:32,212] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
